docs/developer/eventing/sources/kafka-source/README.md

```diff
@@ -13,63 +6,80 @@ Tutorial on how to build and deploy a `KafkaSource` event source.
 
 The `KafkaSource` reads all the messages, from all partitions, and sends those messages as CloudEvents via HTTP to its configured `sink`.
 
-*NOTE:* In case you need a more sophisticated Kafka Consumer, with direct access to specific partitions or offsets you might want to implement a _Kafka Consumer_, using one of the available Apache Kafka SDKs, to handle the messages yourself, rather than using the Knative `KafkaSource`.
+!!! note
+    If you need a more sophisticated Kafka Consumer, with direct access to specific partitions or offsets, you can implement a Kafka Consumer by using one of the available Apache Kafka SDKs.
 
-## Prerequisites
+<!--TODO: Check if this note is out of scope; should we not mention anything beyond the direct Knative/Kafka integration we provide?-->
 
-- A Kubernetes cluster with [Knative Kafka Source installed](../../../admin/install/).
+## Installing the Kafka event source CRD
 
-## Apache Kafka Topic (Optional)
+Install the Kafka event source by running the command:
 
-1. If using Strimzi, you can set a topic modifying `source/kafka-topic.yaml`
-   with your desired:
+```bash
+kubectl apply -f https://storage.googleapis.com/knative-nightly/eventing-kafka/latest/source.yaml
```

Comment:
```suggestion
kubectl apply -f https://github.com/knative-sandbox/eventing-kafka/releases/download/$(curl -sL "https://api.github.com/repos/knative-sandbox/eventing-kafka/releases/latest" | grep "tag_name" | sed -E "s/.*\"tag_name\": \"(.*)\".*/\1/")/source.yaml
```

You can put some verbiage to download and install the latest version from https://github.com/knative-sandbox/eventing-kafka/releases
If you are in a bash shell that contains curl and grep, you can use the following command to install the latest version, I provided the command

Reply:
<opinion>
Those one liners may be fancy in bash scripts, but documentation, I'll take a short explanation over a complicated command anytime.
E.g. "substitute <version> with the current Knative Eventing release version (example `v0.25`)"
</opinion>

Reply:
@csantanapr @abrennan89 apparently the moving "latest" is already solved. I'm seeing a few references such as this one in the docs:

```
kubectl apply -f {{< artifact org="knative-sandbox" repo="eventing-natss" file="300-natss-channel.yaml" >}}
```

Not sure what plugin interprets that, but it looks really cool.

Reply:
On second look, I removed this command altogether, since we are linking to the docs for installing Kafka event source for now